indico/modules/events/editing/schemas.py

```diff
@@ -38,7 +38,15 @@ class RevisionState(mm.Schema):
 class EditingFileTypeSchema(mm.ModelSchema):
     class Meta:
         model = EditingFileType
-        fields = ('id', 'name', 'extensions', 'allow_multiple_files', 'required', 'publishable')
+        fields = ('id', 'name', 'extensions', 'allow_multiple_files', 'required', 'publishable', 'is_used')
+
+    is_used = fields.Function(lambda file_type: EditingRevisionFile.query.with_parent(file_type).has_rows())
+
+    @post_dump(pass_many=True)
+    def sort_list(self, data, many, **kwargs):
+        if many:
+            data = sorted(data, key=lambda e: natural_sort_key(e['name']))
```

Comment:
What does `e` stand for?

I'd go for `x` as a generic name, or `ft` for filetype

Reply:
for me `e` as an entry, but I see no problem with using `ft`